weave-js/src/components/Panel2/PanelPlot/PanelPlot.tsx

```diff
@@ -936,2851 +544,37 @@ const PanelPlotConfigInner: React.FC<PanelPlotProps> = props => {
-      table.columnNames,
-      table.columnSelectFunctions,
-      key,
-      opStore
-    )
-  );
-}
-
-function filterTableNodeToContinuousSelection(
-  node: Node,
-  colId: string,
-  table: TableState.TableState,
-  domain: ContinuousSelection,
-  opStore: OpStore
-): OutputNode {
-  return opFilter({
-    arr: node,
-    filterFn: constFunction({row: listObjectType(node.type)}, ({row}) => {
-      const colName = escapeDots(
-        TableState.getTableColumnName(
-          table.columnNames,
-          table.columnSelectFunctions,
-          colId,
-          opStore
-        )
-      );
-
-      let colNode = opPick({obj: row, key: constString(colName)});
-
-      const domainDiff = domain[1] - domain[0];
-      if (isAssignableTo(colNode.type, maybe(timestampBin))) {
-        return opAnd({
-          lhs: opNumberGreaterEqual({
-            lhs: opDateToNumber({
-              date: opPick({obj: colNode, key: constString('start')}),
-            }),
-            rhs: constNumber(
-              domain[0] - DOMAIN_DATAFETCH_EXTRA_EXTENT * domainDiff
-            ),
-          }),
-          rhs: opNumberLessEqual({
-            lhs: opDateToNumber({
-              date: opPick({obj: colNode, key: constString('stop')}),
-            }),
-            rhs: constNumber(
-              domain[1] + DOMAIN_DATAFETCH_EXTRA_EXTENT * domainDiff
-            ),
-          }),
-        });
-      } else if (isAssignableTo(colNode.type, maybe(numberBin))) {
-        return opAnd({
-          lhs: opNumberGreaterEqual({
-            lhs: opPick({obj: colNode, key: constString('start')}),
-            rhs: constNumber(
-              domain[0] - DOMAIN_DATAFETCH_EXTRA_EXTENT * domainDiff
-            ),
-          }),
-          rhs: opNumberLessEqual({
-            lhs: opPick({obj: colNode, key: constString('stop')}),
-            rhs: constNumber(
-              domain[1] + DOMAIN_DATAFETCH_EXTRA_EXTENT * domainDiff
-            ),
-          }),
-        });
-      }
-      if (
-        isAssignableTo(
-          colNode.type,
-          maybe({
-            type: 'timestamp',
-            unit: 'ms',
-          })
-        )
-      ) {
-        colNode = opDateToNumber({date: colNode});
-      }
-
-      return opAnd({
-        lhs: opNumberGreaterEqual({
-          lhs: colNode,
-          rhs: constNumber(
-            domain[0] - DOMAIN_DATAFETCH_EXTRA_EXTENT * domainDiff
-          ),
-        }),
-        rhs: opNumberLessEqual({
-          lhs: colNode,
-          rhs: constNumber(
-            domain[1] + DOMAIN_DATAFETCH_EXTRA_EXTENT * domainDiff
-          ),
-        }),
-      });
-    }),
-  });
-}
-/*
-
-async function mergeRealizedTableData(
-  oldTable: _.Dictionary<any>[],
-  newTable: _.Dictionary<any>[]
-): Promise<_.Dictionary<any>[]> {
-  const flatArray = _.flatten([oldTable, newTable]);
-  const uniqueArray = _.uniqBy(flatArray, '_rowIndex');
-  return _.sortBy(uniqueArray, '_rowIndex');
-}
-*/
```

Comment:
We should keep this in a commented out form in case we want to re-enable it later on